### ISSUE TITLE
[Discussion] Consider recommending Libreddirect instead of Privacy Redirect

### ISSUE THREAD

**Issue by TheFrenchGhosty** [iv-org/invidious#2896]
**Half enhancement, Half discussion**

We're looking for user opinions.

**Is your enhancement request related to a problem? Please describe.**

We currently endorse, and even recommend Privacy Redirect, however the project is basically dead, and unmaintained.

**Describe the solution you'd like**

We can instead recommend Libreddirect: https://github.com/libredirect/libredirect

**Describe alternatives you've considered**

We take over (or fork) Privacy Redirect, this is not a really good idea considering we already have stuff to work on, and I don't think anyone in the team is a "good" JS developer.

**Additional context**

Libreddirect has multiples issue:

@SimonBrazell is against the fork: https://github.com/SimonBrazell/privacy-redirect/pull/319#issuecomment-987361792

No one currently uses it (15 users according to https://addons.mozilla.org/en-US/firefox/addon/libredirect/ )

We had a discussion if [the offtopic room](https://matrix.to/#/!BUFmfOtsijlOMmJjDg:pussthecat.org/$gyoxMU1zmp-lyHfALh4rDyKyEHuoqgqLCF4RXOe1giE?via=pussthecat.org&via=matrix.org&via=tchncs.de), some useful information/points:

![1](https://user-images.githubusercontent.com/47571719/153760355-3a06dc6c-7bbd-42f5-a079-ef792b2c60ff.png)
![2](https://user-images.githubusercontent.com/47571719/153760360-3a14d162-bc37-4e0f-8917-73f5a53ba793.png)

**Comment by IronMaltese** [comment]
I completely agree, since Libredirect is actively maintained with a lot more features and more frontend redirects than Privacy Redirect, so I think this should be implemented

**Comment by ManeraKai** [comment]
> @SimonBrazell is against the fork: https://github.com/SimonBrazell/privacy-redirect/pull/319#issuecomment-987361792

Many of the changes we made won't make @SimonBrazell happy. I'll just fight with him a lot even about the code organization and structuring scheme. He's also busy and there's no time to wait for an approval to a PR. Either he let me be an admin on PrivacyRedirect (which is just an occupation of his own project. He also doesn't really know me so he won't be able to trust me) or fork it with peace and everyone takes his path.
As I'm adding and committing new fixes and features to LibRedirect, I really don't regret that we forked it as it was the right choice.

> No one currently uses it (15 users according to https://addons.mozilla.org/en-US/firefox/addon/libredirect/ )

Well today my friend we've exceeded 1k active users on Firefox! https://addons.mozilla.org/en-US/firefox/addon/libredirect/ and it's still growing. You know the feeling when you see your child grow 🥰
